vhd/packer/configure-windows-vhd.ps1

```diff
@@ -1,349 +1,349 @@
-<#

```

Comment:
Ugh, GH made this a whitespace diff when I merged changes from another PR. Let me know if this is too hard to grok and I'll re-rebase.